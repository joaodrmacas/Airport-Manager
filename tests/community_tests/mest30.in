a LIS Portugal Lisbon
a FAO Portual FAO
v TP12 LIS FAO 23-03-2022 11:00 00:45 10
r TP12 23-03-2022 GT3GT3GT33 6
r TP12 23-03-2022 GT4GT4GT44 8
r TP12 23-03-2022 GT4GT4GT44 4
r TP12 23-03-2022
v
e GT3GT3GT33
r TP12 23-03-2022
v
e TP12
v
q
