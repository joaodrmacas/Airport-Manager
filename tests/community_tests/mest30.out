airport LIS
airport FAO
too many reservations
GT3GT3GT33 6
GT4GT4GT44 4
TP12 LIS FAO 23-03-2022 11:00
GT4GT4GT44 4
TP12 LIS FAO 23-03-2022 11:00
